pkgs/servers/sip/freeswitch/default.nix

```diff
@@ -110,6 +126,15 @@ stdenv.mkDerivation rec {
     done
   '';
 
+  ## TODO Validate with the next upstream release
+  pathces = [
```

Comment:
```suggestion
  patches = [
```

Reply:
Unless I'm missing something this change is still needed.

Reply:
You're correct.